how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -1,64 +1,82 @@
 # How to use Kata Containers and CRI (containerd plugin) with Kubernetes
 
-This document describes how to set up a single-machine Kubernetes cluster. 
-The Kubernetes cluster will use the CRI containerd plugin and Kata Containers to launch untrusted workloads.
+* [Requirements](#requirements)
+* [Install containerd with CRI plugin enabled](#install-containerd-with-cri-plugin-enabled)
+* [Install Kata Containers](#install-kata-containers)
+* [Install Kubernetes](#install-kubernetes)
+* [Configure containerd to use Kata Containers](#configure-containerd-to-use-kata-containers)
+    * [Define the Kata runtime as the untrusted workload runtime](#define-the-kata-runtime-as-the-untrusted-workload-runtime)
+* [Configure Kubelet to use containerd](#configure-kubelet-to-use-containerd)
+* [Configure proxy - OPTIONAL](#configure-proxy---optional)
+* [Start Kubernetes](#start-kubernetes)
+* [Install a Pod Network](#install-a-pod-network)
+* [Allow pods to run in the master node](#allow-pods-to-run-in-the-master-node)
+* [Create an unstrusted pod using Kata Containers](#create-an-unstrusted-pod-using-kata-containers)
+* [Delete created pod](#delete-created-pod)
+
+This document describes how to set up a single-machine Kubernetes (k8s) cluster.
+
+The Kubernetes cluster will use the
+[CRI containerd plugin](https://github.com/containerd/cri) and
+[Kata Containers](https://katacontainers.io) to launch untrusted workloads.
+
+## Requirements
 
-## Requirements 
 - Kubernetes, kubelet, kubeadm
 - cri-containerd
 - Kata Containers
 
-For information about the supported version of these components see 
-Kata Containers [versions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml) file.
+**Note:** For information about the supported versions of these components,
```

Comment:
Should this be an `>` indented note as per our [guidelines](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes)